code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -18,6 +18,7 @@
 	volume = 250
 	possible_transfer_amounts = null
 	var/delay = CLICK_CD_RANGE * 2
+	var/no_refill = FALSE
```

Comment:
I don't think we should need this here, we have some container flags which should define whether or not we can refill them (see something like OPENCONTAINER).

Reply:
Did you talk about this in the discord? Or is there another reason that this was resolved?

Reply:
I see, review got applied 👍